### DISCUSSION TITLE
Creating related many-to-many items in graphql

### DISCUSSION THREAD

**Discussion by krazyjakee** [directus/directus#9770] [Archive]
The docs have a [single snippet for creating related items in a query](https://docs.directus.io/reference/introduction/#graphql) but it's incomplete and unclear how to apply this to a mutation.

Here is what I have that feels like it should be working but isn't...

```graphql
mutation {
  create_hardware_item(data: {
    name: "test",
    videos: [{url:"http",status:"Published"}]
  }
}
```

The error thrown is `Field "url" is not defined by type "create_hardware_videos_input".",`. The url is certainly a field on the videos collection so why would it say that?

Any help would be appreciated, thanks.

**Comment by azrikahar** [comment]
I believe the snippet linked should only be applicable to queries, not mutations, at least at the moment. The issue you are facing should be #7237 👍
